examples/chat/build.gradle

```diff
@@ -7,5 +7,5 @@ plugins {
 apply from: "../examples.gradle"
 
 dependencies {
-    implementation("com.graphql-java:graphql-java-extended-scalars:1.0.1")
+    implementation("com.graphql-java:graphql-java-extended-scalars:2020-02-19T04-04-26-6f82be9")
```

Comment:
@alvarosanchez it seems this https://github.com/micronaut-projects/micronaut-graphql/blob/master/graphql/build.gradle#L14-L18 isn't working

Reply:
Forget about it. I found the issue, the dependency is from one of the examples, not the main project.

Reply:
If I'd only knew how to read... :man_facepalming: 

![image](https://user-images.githubusercontent.com/559192/87011588-6926ec80-c1c8-11ea-8a4a-fdf30ab40769.png)